data_analysis/pii/detect_secret.ipynb

```diff
@@ -0,0 +1,268 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "6a553e5c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "16\n",
+      "oppia/tests/av/test_permissions.py\n",
+      "[{'type': 'Secret Keyword', 'line_number': 15, 'secret_value': 'password'}]\n",
+      "\n",
+      "47\n",
+      "tests/openwisp/test_backend.py\n",
+      "[{'type': 'Private Key', 'line_number': 101, 'secret_value': 'BEGIN RSA PRIVATE KEY'}]\n",
+      "\n",
+      "48\n",
+      "tests/admin_views/tests.py\n",
+      "[{'type': 'Secret Keyword', 'line_number': 81, 'secret_value': 'secret'}, {'type': 'Secret Keyword', 'line_number': 4104, 'secret_value': 'newpassword'}, {'type': 'Secret Keyword', 'line_number': 1095, 'secret_value': 'notsecret'}, {'type': 'Secret Keyword', 'line_number': 4129, 'secret_value': 'mismatch'}, {'type': 'Secret Keyword', 'line_number': 4908, 'secret_value': 'test'}]\n",
+      "\n",
+      "53\n",
+      "server/stocktickerneo.py\n",
+      "[{'type': 'Secret Keyword', 'line_number': 17, 'secret_value': 'password'}, {'type': 'Basic Auth Credentials', 'line_number': 35, 'secret_value': '%s'}]\n",
+      "\n",
+      "76\n",
+      "bts_tools/cmdline.py\n",
+      "[{'type': 'Base64 High Entropy String', 'line_number': 124, 'secret_value': '-DCMAKE_BUILD_TYPE=RelWithDebInfo'}]\n",
+      "\n",
+      "89\n",
+      "pootle/auth/ldap_backend.py\n",
+      "[{'type': 'Secret Keyword', 'line_number': 70, 'secret_value': 'LDAP_%s'}]\n",
+      "\n",
+      "100\n",
+      "stages/keystone/fabfile.py\n",
+      "[{'type': 'Basic Auth Credentials', 'line_number': 50, 'secret_value': '%s'}]\n",
+      "\n",
+      "135\n",
+      "docs/plans/big-data/testing.py\n",
+      "[{'type': 'Hex High Entropy String', 'line_number': 237, 'secret_value': '2988c44780790e4ffceb1f97391e475f165e316f27a656957282a2998aee9d4f'}, {'type': 'Hex High Entropy String', 'line_number': 236, 'secret_value': 'fa4e6aa7e7edaa6feb036fd5e8c28ffc48575cefc332187552c5be4bf0511af8'}, {'type': 'Hex High Entropy String', 'line_number': 245, 'secret_value': '46dabc02b59be44064a9e06dd50bc6841833578c2b6339fbc43f090cc17831fa'}, {'type': 'Hex High Entropy String', 'line_number': 248, 'secret_value': '9b4a551a6c1c5830d6070b9c22ae1788b9743e9637be47d56103bcda019a897c'}, {'type': 'Hex High Entropy String', 'line_number': 229, 'secret_value': '374c02504f89ed0a760b03c3e1fd2258988576b919d763254709b66fc7bfb22e'}, {'type': 'Hex High Entropy String', 'line_number': 228, 'secret_value': 'd07050610c50de8c810aa1d1e322786ed8932cf6eafa0fbe1f132b2c881af9c2'}, {'type': 'Hex High Entropy String', 'line_number': 249, 'secret_value': '9820f1ab795db7b0d195f21966ecb071d76e9ce2fd3a90845974a3905584eb3e'}, {'type': 'Hex High Entropy String', 'line_number': 244, 'secret_value': '983730afb7ab41d524b72f1097daaf4a3c15b98943291f96e523730849cabe8c'}]\n",
```

Comment:
Seems like there are a few false positives in these results. Is this intentional?